dora/tests/integration/src/test/java/alluxio/client/fs/DoraFileSystemIntegrationTest.java

```diff
@@ -81,6 +86,7 @@ public final class DoraFileSystemIntegrationTest extends BaseIntegrationTest {
           .setProperty(PropertyKey.WORKER_HTTP_SERVER_ENABLED, false)
           .setProperty(PropertyKey.S3A_ACCESS_KEY, mS3Proxy.getAccessKey())
           .setProperty(PropertyKey.S3A_SECRET_KEY, mS3Proxy.getSecretKey())
+          .setProperty(PropertyKey.DORA_CLIENT_UFS_FALLBACK_ENABLED, false)
```

Comment:
Is this an intended change? Or it's just a test run to expose issues?

Reply:
Yes, it is an intended change. If we want to test DoraFileSystem, we have to disable the Ufs Fallback, otherwise the methods like `createFile` `getStatus` will fall back to Ufs, while we want to test DoraFileSystem.

Reply:
@dbw9580 PTAL. I'm fine with disabling fallback by default, but I'd like to make sure that if a cluster sets `fallback=true`, that will also work.